design/BulkAPI.md

```diff
@@ -28,18 +28,23 @@ progress of the job.
 {
   "filter": {
     "exclude": {
-      "namespace": [],
-      "workload": [],
-      "containers": [],
-      "labels": {}
+      "namespace": ["cadvisor", "openshift-tuning", "openshift-monitoring", "thanos-bench"],
+      "workload": ["osd-rebalance-infra-nodes-28887030", "blackbox-exporter", "thanos-query"],
+      "containers": ["tfb-0", "alertmanager"],
+      "labels": {
+        "org_id": "ABCOrga",
+        "source_id": "ZZZ",
+        "cluster_id": "ABG"
+      }
     },
     "include": {
-      "namespace": [],
-      "workload": [],
-      "containers": [],
+      "namespace": ["cadvisor", "openshift-tuning", "openshift-monitoring", "thanos-bench"],
+      "workload": ["osd-rebalance-infra-nodes-28887030", "blackbox-exporter", "thanos-query"],
+      "containers": ["tfb-0", "alertmanager"],
```

Comment:
This is a bad example, the same things are present in both the `include` and `exclude` criteria. Also are not supporting wildcard chars here?

What we need to support is something like this
```
"exclude": {
   "namespace": ["openshift-*"]
},
"include": {
   "namespace": ["openshift-tuning"]
}
```

This should exclude all "openshift-" namespaces except "openshift-tuning"